assets/js/keyboard-shortcuts.js

```diff
@@ -34,6 +35,11 @@ export const keyboardShortcuts = [
     hasModifier: true,
     action: searchKeyAction
   },
+  {
+    key: 'v',
+    description: 'Open/focus version select',
```

Comment:
We could use `'showPicker' in HTMLSelectElement.prototype` to determine the language.